src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/erasure-code-profile-form/erasure-code-profile-form-modal.component.html

```diff
@@ -104,14 +116,18 @@
             <span class="invalid-feedback"
                   *ngIf="form.showError('m', frm, 'min')"
                   i18n>Must be equal to or greater than 1.</span>
+            <span class="invalid-feedback"
+                  *ngIf="form.showError('m', frm, 'max')"
+                  i18n>Coding chunks have exceeded the available OSDs of {{deviceCount}}.</span>
```

Comment:
Can't we change this to 

```suggestion
                  i18n>Chunks (k+m) have exceeded the available OSDs of {{deviceCount}}.</span>
```

same for the Data Chunks.
Otherwise the error message is misleading:
![image](https://user-images.githubusercontent.com/399326/80479115-14503580-893e-11ea-8ec1-91daa91409f4.png)

Reply:
Sure and done :)